code/__HELPERS/_macros.dm

```diff
@@ -39,7 +39,7 @@
 
 #define iscluwne(A) istype(A, /mob/living/simple_animal/hostile/retaliate/cluwne)
 
-#define isAI(A) istype(A, /mob/living/silicon/ai)
+#define isAI(A) (istype(A, /mob/living/silicon/ai) || (isobserver(A) && ticker.current_state == GAME_STATE_FINISHED)) 
```

Comment:
@clusterfack I want them to be counted as AI, so, I make isAI also return true of the mob is an observer and the game is finished.